### PR TITLE
Reorder error steps in request PiP

### PR DESCRIPTION
**Pull request by beaufortfrancois**
I'm reordering error steps so that web developer can take much easier some actions on this.

1. PiP is not enabled (you can't do anything)
2. Feature Policy prevents you to use PiP (update it if you can)
3. Video attribute prevents you to use PiP (remove it)

### CODE REVIEW

File: index.bs [hunk 1]
```diff
@@ -277,9 +277,9 @@ partial interface Document {
 </pre>
 
 The {{pictureInPictureEnabled}} attribute's getter must return
-<code>true</code> if the <a>context object</a> is <a>allowed to use</a> the
-feature indicated by attribute name <code>picture-in-picture</code> and
-<a>Picture-in-Picture is supported</a>, and false otherwise.
+<code>true</code> if <a>Picture-in-Picture is supported</a> and the <a>context
+object</a> is <a>allowed to use</a> the feature indicated by attribute name
+<code>picture-in-picture</code>, and false otherwise.
```
Comment (mounirlamouri) [inline index.bs]: Isn't this about the feature policy? Does that mean that the steps above (checking for `pictureInPictureEnabled` would check for the feature policy too implicitly, making step 2 moot?

Comment (beaufortfrancois) [reply index.bs]: You're absolutely right! Good catch.

**Comment by beaufortfrancois** [comment]
@mounirlamouri Feedback addressed. WDYT?

File: index.bs [hunk 2]
```diff
@@ -277,12 +277,14 @@ partial interface Document {
...
-
-<dfn>Picture-in-Picture is supported</dfn> if there is no
-previously-established user preference, security risk, or platform limitation.
+<code>true</code> if <a>Picture-in-Picture support</a> is <code>true</code> and
+the <a>context object</a> is <a>allowed to use</a> the feature indicated by
+attribute name <code>picture-in-picture</code>, and <code>false</code>
+otherwise.
+
+<dfn>Picture-in-Picture support</dfn> is <code>true</code> if there is no
+previously-established user preference, security risk, or platform limitation,
```
Comment (mounirlamouri) [inline index.bs]: security risk? That's unclear. Restrictions? Though maybe keep to "platform limitation"?

**Pull request merged**